flink-libraries/flink-table/src/main/scala/org/apache/flink/table/expressions/fieldExpression.scala

```diff
@@ -191,12 +174,10 @@ case class RowtimeAttribute(expr: Expression) extends TimeAttribute(expr) {
         ValidationSuccess
       case WindowReference(_, _) =>
         ValidationFailure("Reference to a rowtime or proctime window required.")
-      case TemporalTableReference(_, _) =>
-        ValidationSuccess
       case any =>
         ValidationFailure(
-          s"The '.rowtime' expression can only be used for table definitions, windows, " +
-            s" and temporal tables, while [${any}] was found.")
+          s"The '.rowtime' expression can only be used for table definitions, windows " +
+            s"and temporal table definitions, while [$any] was found.")
```

Comment:
I think table definitions are not supported anymore. So we can reset this message entirely, right?

Reply:
Yes, you are right :)